file_util.go

```diff
@@ -98,8 +117,10 @@ func (util *snowflakeFileUtil) getDigestAndSizeForFile(fileName string) (string,
 		total += int64(n)
 		m.Write(chunk[:n])
 	}
-	f.Seek(0, io.SeekStart)
-	return base64.StdEncoding.EncodeToString(m.Sum(nil)), total, nil
+	if _, err = f.Seek(0, io.SeekStart); err != nil {
+		return "", -1, err
+	}
+	return base64.StdEncoding.EncodeToString(m.Sum(nil)), total, err
```

Comment:
if `err != nil` we won't reach this code, so might as well return `nil` for clarity

Reply:
Ha, definitely not :) This is something called naked return - we name variables that are returned. Without that, if we rich this line 123, err set in a deferred block would be swallowed.

Reply:
Oh, that makes a lot of sense. I stand corrected 😄